src/shared/components/syntax-highlighter/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Prism from 'prismjs';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import 'prismjs/themes/prism-okaidia.css';
+import styles from './index.module.css';
+
+const SyntaxHighlighter = ({ codeStr }) => {
+    const htmlCode = Prism.highlight(codeStr, Prism.languages.js, 'js');
+    const preClasses = classNames('language-js', styles.preCustom);
+
+    return (
+        <div className={ styles.gatsbyHighlight } >
```

Comment:
Do you really need this outer div?

Reply:
Yes, I do. This way, I have greater specificity to apply `css` rules